File: remove_duplicates_from_sorted_array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
                    
        unique = 0
        for i in range(0, len(nums)):
            if nums[unique] != nums[i]:
                unique += 1
                nums[unique] = nums[i]

        return (unique+1)

File: remove_duplicates_from_sorted_array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    
    let j=0;

    for (let i=0; i < nums.length; i++){
        
        if (nums[i] != nums[j]){
            nums[j+1] = nums[i];
            j++;
        }
    }
    return j+1;
};